content/300-guides/300-upgrade-guides/200-upgrading-versions/050-upgrading-to-prisma-3/index.mdx

```diff
@@ -40,6 +40,64 @@ This means that if your application relied on `$queryRaw` calls using _strings_,
 
 You can learn more about the new `$queryRaw` and `$queryRawUnsafe` methods in the [Raw database access](../../../../concepts/components/prisma-client/raw-database-access) section of the docs.
 
+### [Json Null Equality](../../../../concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-by-null-values)
+
+We recently learned from [a community member](https://github.com/prisma/prisma/issues/8399) that you cannot filter a `Json` field by a null value. This is because `{ equals: null }` checks if the column value in the database is `NULL`, not if the JSON value inside the column equals `null`.
```

Comment:
```suggestion
You cannot filter a `Json` field by a null value.  [See this GitHub issue](https://github.com/prisma/prisma/issues/8399). 
This is because `{ equals: null }` checks if the column value in the database is `NULL`, not if the JSON value inside the column equals `null`.
```